.github/workflows/CompileOrRun.yml

```
@@ -87,7 +87,7 @@ jobs:
           using Pkg
           Pkg.Registry.update()
           # Pkg.add([
-          #     PackageSpec(; name="Reactant", rev="main"),
+          #     PackageSpec(; name="Reactant", rev="ap/fix_type"),
```

```suggestion
          #     PackageSpec(; name="Reactant", rev="main"),
```
but the fix isn't in any registered version yet.

---

ok